docs/shim/instrumentation.rst

```diff
@@ -11,6 +11,3 @@ Submodules
 
 .. toctree::
     :maxdepth: 1
-
-    instrumentor
```

Comment:
not sure if this should have been here in the first place but it seems unrelated to metrics